src/syscheckd/db/src/file.cpp

```diff
@@ -110,7 +110,8 @@ FIMDBErrorCode fim_db_remove_path(const char* path)
     {
         try
         {
-            const auto removeFileCondition { std::string("WHERE path=") + std::string(path) };
+            nlohmann::json removeFileCondition;
+            removeFileCondition["path"] = std::string(path);
```

Comment:
is std::string needed ?
```suggestion
            removeFileCondition["path"] = path;
```

Reply:
You are right ! This is a redundancy, I am changing it